lib/main.dart

```diff
@@ -1,113 +1,151 @@
+                      boxShadow: [
+                        BoxShadow(
+                          blurRadius: Constants.avatarBlurShadow,
+                          color: Constants.avatarShadowColor,
+                          spreadRadius: 1,
+                        ),
+                      ],
+                    ),
+                    child: CircleAvatar(
+                      backgroundImage: AssetImage(
+                        'assets/images/me.jpg',
+                      ),
+                      radius: Constants.avatarRadius,
+                    ),
+                  ),
+                  SizedBox(
+                    height: Constants.avatarAndPersonNameSizeBoxHeight,
+                  ),
+                  Text(
+                    'Kevin Guillen',
+                    style: TextStyle(
+                      fontSize: Constants.personNameFontSize,
+                      fontWeight: FontWeight.bold,
+                      color: Constants.personNameTextColor,
+                      letterSpacing: Constants.letterSpacing,
+                    ),
+                  ),
+                  SizedBox(
+                    height: Constants.personNameAndPositionSizeBoxHeight,
+                  ),
+                  Text(
+                    'Flutter Developer Trainee',
+                    style: TextStyle(
+                      fontSize: Constants.positionFontSize,
+                      fontWeight: FontWeight.bold,
+                      color: Constants.positionTextColor,
+                      letterSpacing: Constants.letterSpacing,
+                    ),
+                  ),
+                  Column(
+                    children: [
+                      Divider(
+                        height: 60,
+                        thickness: 3,
+                        color: Colors.blueAccent.shade100,
+                        indent: 60,
+                        endIndent: 60,
+                      ),
+                    ],
+                  ),
+                  Card(
+                    color: Constants.cardsBackgroundColor,
+                    child: ListTile(
+                      leading: Padding(
+                        padding: EdgeInsets.only(
+                          top: 8.0,
+                        ),
+                        child: Icon(
+                          Icons.phone,
+                          color: Constants.phoneIconColor,
+                        ),
+                      ),
+                      title: Text(
+                        '+54 9 11 2494 607181',
+                        style: TextStyle(
+                          fontSize: Constants.phoneNumberFontSize,
+                          letterSpacing: Constants.letterSpacing,
+                          color: Constants.contactInfoTextColor,
+                        ),
+                      ),
+                      subtitle: Text(
+                        "Cell Phone",
+                        style: TextStyle(
```

Comment:
For the future, you could create TextStyles to be used all around your app, instead of having to create each of them every time, for example, `titleTextStyle` which will have the standard style for the texts which are headers or titles ?

Reply:
done